molecule/default/playbook.yml

```diff
@@ -13,6 +13,9 @@
 
     - role: ansible-role-prometheus-postgres
       prometheus_postgres_dbname: alice
+      prometheus_postgres_data_source_name: user=postgres dbname=
+        {{ prometheus_postgres_dbname }} host=/var/run/postgresql/
+        sslmode=disable
```

Comment:
```suggestion
```

Reply:
I should keep the line being breaked in 3 lines, otherwise, travis CI telling me there is a line contains more than 80 characters